script/Ciel/Day 13/13_01_01_CIEL13_5B.txt

```diff
@@ -101,42 +102,42 @@ That's right. Please don't get yourself involved with me. Although I may act lik
 [sha:9ebc699e8c774f6a0d504d5c2601d7491ad40987]{
 -- Page 573, Offset 34916.
 -- 彼は手にした紙袋からきちんと包装された小箱を取り出した。
-The thing he pulls out of the paper bag is a small, tightly-wrapped box.
+He pulls out a small, tightly-wrapped box from the paper bag in his hands.
 }
 [sha:20bfde6e766718d53c56b1976a290d217dc797cf]{
 -- Page 573, Offset 34917.
 -- 間違いない。あれはわたしが欲しがっていた銀製のアクセサリ－。十字架とベルを兼ねた、中学生にはまだ早い<ま|・><せ|・><た|・>品物だ。
-There's no doubt about it. That's the silver accessory that I've always wanted. An item that serves both as a cross and a bell; something too mature for a junior high-school student.
+There's no doubt about it. That's the silver accessory that I've always wanted. It doubles as both a bell and a cross; an ornament too mature for a middle school student like me.
```

Comment:
if there's a cg showing what the accessory actually is, we should specify. I'm assuming it's a necklace but if not, it can stay "accessory". That said, "always" doesn't really make sense.
> That's the silver necklace I've been wanting.